.vscode/tasks.json

```diff
@@ -27,7 +27,18 @@
                 "kind": "build",
                 "isDefault": true
             },
-            "problemMatcher": ["$tsc-watch", "$ts-checker-webpack-watch"]
+            "problemMatcher": ["$ts-webpack-watch"]
+        },
+        {
+            "label": "Compile Web Extension",
+            "type": "npm",
+            "script": "compile-web-watch",
```

Comment:
If you want to test/run any of the web version of stuff, you can run this task now.

It's more finicky than normal ts though. Sometimes errors it generates aren't shown in the problems window.

Reply:
Note how this is different than building the webview-ui parts. Those are separate bundles. For now. I might have to include them when I get any of the webview UI building